apps/site/assets/css/_schedule-page.scss

```diff
@@ -379,7 +379,7 @@
   border-top: 3px solid $gray-lightest;
   display: flex;
   justify-content: space-around;
-  // scss-lint:disable DuplicateProperty IE11 doesn't support space-evenly
+  // IE11 doesn't support space-evenly
```

Comment:
This seems like it should be one line up, since it's really describing the `space-around` rule.